src/builtins/BuiltinAtomics.cpp

```diff
@@ -595,7 +464,7 @@ static Value builtinAtomicsNotify(ExecutionState& state, Value thisValue, size_t
 
 static Value builtinAtomicsIsLockFree(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
 {
-    auto size = argv[0].toInteger(state);
+    const size_t size = static_cast<size_t>(argv[0].toInteger(state));
```

Comment:
`toInteger`  return `double` type, Is it safety on both 64-bit and 32-bit machines?

Reply:
Good catch! Its my mistake. I fixed it to double typed value.